src/components/ErrorModal.tsx

```diff
@@ -0,0 +1,41 @@
+import Button from '@/design-system/inputs/Button'
+import Modal from '@/design-system/modals/Modal'
+import React from 'react'
+import Link from './Link'
+import Trans from './translation/Trans'
+
+interface ErrorModalProps {
+  error: Error | null
+}
+
+const ErrorModal: React.FC<ErrorModalProps> = () => {
```

Comment:
Ça manque de petite animation tout ça

Reply:
t'as quelque chose en tête ? Le corps d'un astronaute flottant dans l'espace ?

Reply:
You read my mind